src/orquestra/sdk/_base/cli/_dorq/_repos.py

```diff
@@ -107,15 +110,42 @@ class ConfigRepo:
     Wraps accessing ~/.orquestra/config.json
     """
 
-    def list_config_names(self) -> t.Sequence[ConfigName]:
-        config_entries = sdk.RuntimeConfig.list_configs()
-        return [
-            # .list_configs() should probably already return "ray" and "in_process".
-            # TODO: fix that in https://zapatacomputing.atlassian.net/browse/ORQSDK-674
-            _config.RAY_CONFIG_NAME_ALIAS,
-            _config.IN_PROCESS_CONFIG_NAME,
-            *config_entries,
-        ]
+    @staticmethod
```

Comment:
**suggestion**: let's not use static methods in the components related to the CLI. 

We expect the caller to use this as instance method (`obj.list_config_names(...)`, not class method (`ConfigRepo.list_config_names()`) because the latter requires more boilerplate when setting up mocks in isolated tests.